app/src/org/commcare/android/integrity/IntegrityTokenViewModel.kt

```diff
@@ -115,7 +117,7 @@ class IntegrityTokenViewModel(application: Application) : AndroidViewModel(appli
     }
 
     interface IntegrityTokenCallback {
-        fun onTokenReceived(token: String, requestHash: String)
+        fun onTokenReceived(token: String, requestHash: String, integrityTokeResponse: StandardIntegrityManager.StandardIntegrityToken)
```

Comment:
no need to send the `token` separately if you are seding the whole response and implementations can get token directly from the response instead.